src/model/input.ts

```diff
@@ -8,129 +12,212 @@ const core = require('@actions/core');
-    return core.getInput('androidKeyaliasName') || '';
+    return Input.getInput('androidKeyaliasName') || '';
   }
 
   static get androidKeyaliasPass() {
-    return core.getInput('androidKeyaliasPass') || '';
+    return Input.getInput('androidKeyaliasPass') || '';
   }
 
   static get androidTargetSdkVersion() {
     return core.getInput('androidTargetSdkVersion') || '';
   }
 
   static get allowDirtyBuild() {
-    const input = core.getInput('allowDirtyBuild') || false;
+    const input = Input.getInput('allowDirtyBuild') || false;
 
     return input === 'true';
   }
 
   static get customParameters() {
-    return core.getInput('customParameters') || '';
+    return Input.getInput('customParameters') || '';
   }
 
   static get sshAgent() {
-    return core.getInput('sshAgent') || '';
+    return Input.getInput('sshAgent') || '';
+  }
+
+  static async githubToken() {
+    return Input.getInput('githubToken') || (await GithubCliReader.GetGitHubAuthToken()) || '';
   }
 
-  static get gitPrivateToken() {
-    return core.getInput('gitPrivateToken') || '';
+  static async gitPrivateToken() {
+    return core.getInput('gitPrivateToken') || (await Input.githubToken());
   }
 
   static get chownFilesTo() {
-    return core.getInput('chownFilesTo') || '';
+    return Input.getInput('chownFilesTo') || '';
   }
 
-  static get remoteBuildCluster() {
-    return core.getInput('remoteBuildCluster') || '';
+  static get postBuildSteps() {
+    return Input.getInput('postBuildSteps') || '';
   }
 
-  static get awsStackName() {
-    return core.getInput('awsStackName') || '';
+  static get preBuildSteps() {
+    return Input.getInput('preBuildSteps') || '';
   }
 
-  static get kubeConfig() {
-    return core.getInput('kubeConfig') || '';
+  static get customJob() {
+    return Input.getInput('customJob') || '';
+  }
+
+  static get cloudRunnerCluster() {
+    return Input.getInput('cloudRunnerCluster') || '';
+  }
+
+  static get awsBaseStackName() {
+    return Input.getInput('awsBaseStackName') || 'game-ci';
   }
 
-  static get githubToken() {
-    return core.getInput('githubToken') || '';
+  static get kubeConfig() {
+    return Input.getInput('kubeConfig') || '';
   }
 
-  static get remoteBuildMemory() {
-    return core.getInput('remoteBuildMemory') || '800M';
+  static get cloudRunnerMemory() {
+    return Input.getInput('cloudRunnerMemory') || '750M';
   }
 
-  static get remoteBuildCpu() {
-    return core.getInput('remoteBuildCpu') || '0.25';
+  static get cloudRunnerCpu() {
+    return Input.getInput('cloudRunnerCpu') || '1.0';
   }
 
   static get kubeVolumeSize() {
-    return core.getInput('kubeVolumeSize') || '5Gi';
+    return Input.getInput('kubeVolumeSize') || '5Gi';
   }
 
   static get kubeVolume() {
-    return core.getInput('kubeVolume') || '';
+    return Input.getInput('kubeVolume') || '';
+  }
+
+  public static ToEnvVarFormat(input: string) {
+    return input
+      .replace(/([A-Z])/g, ' $1')
+      .trim()
```

Comment:
Note 2: in javascript we use `camelCase` for all definitions except class names which must be `PascalCase`.

Perhaps we should add an ESLint rule that enforces this rule (also fine to just create an issue).

Reply:
Issue exists to address eslint rule